R/utils.R

```diff
@@ -139,3 +139,34 @@ check_package_available <- function(pkg) {
     stop("Please install suggested package: ", pkg)
   }
 }
+
+# hidden, also in hemibrainr
+add_field_seq <- function(x, entries, field = "bodyid", ...){
+  x = nat::as.neuronlist(x)
+  if(length(entries)!=length(x)){
+    stop("The length of the entries to add must be the same as the length of the neuronlist, x")
+  }
+  nl = nat::neuronlist()
+  for(i in 1:length(x)){
+    y = x[[i]]
+    entry = entries[i]
+    y[[field]] = entry
+    nl = c(nl, nat::as.neuronlist(y))
+  }
+  names(nl) = names(x)
+  nl[,] = x[,]
+  nl
+}
+
+# hidden
+nullToZero <- function(x) {
```

Comment:
NB there is already an internal null2na function here https://github.com/natverse/fafbseg/blob/257e2372ebb59e58b008c6da62466bfe30ee5c45/R/ids.R#L323. Maybe we need a `replace_nulls` function with the option to use different replacement values and a consistent interface.

Reply:
yes, that would be the best outcome